CONTRIBUTING.md

```diff
@@ -52,7 +52,7 @@ Once you have babel running in watch mode, you can start making changes to parse
     * **Windows**: `npm run coverage:win`
     * **Unix**: `npm run coverage`
 * Run the tests for the file you are working on with the following command:
-    * **Windows**: `npm run test:win spec/MyFile.spec.js`
+    * **Windows**: `npm run test spec/MyFile.spec.js`
     * **Unix**: `npm test spec/MyFile.spec.js`
```

Comment:
`test:win` doesn't exist in` package.json` so it can be removed. 

Since windows and unix run the same command you can clean this up. 

The same can be done for `coverage:win`

Reply:
ok, done